### DISCUSSION TITLE
Does yazi make a local cache if I use it over SSH?

### DISCUSSION THREAD

**Discussion by cbr9** [sxyazi/yazi#1868] [1. Q&A]
### What system are you running Yazi on?

Linux X11

### What terminal are you running Yazi in?

Windows Terminal

### `yazi --debug` output

```Shell
Not applicable
```

### Describe the question

I want to know if Yazi caches files locally. I want to use it at work but I cannot if it makes a local cache, since that might download customer data.

If it does, is there a way to disable it?

### Validations

- [x] I have searched the existing discussions/issues

**Comment by sxyazi** [comment]
If by "cache" you mean "thumbnail cache" and you're running Yazi on the server, then Yazi will only generate it **on the server** (on Linux, it's saved under `/tmp/yazi-{uid}`), and you can disable it anytime in your `yazi.toml`:

```toml
[plugin]
preloaders = []
```
